src/program.js

```diff
@@ -246,6 +246,14 @@ Example: $0 --help run.
       describe: 'Show verbose output',
       type: 'boolean',
     },
+    'ignore-files': {
```

Comment:
This comment is supposed to be part of the last review comments but it has not been submitted by github, and so I added it here as a separate comment.

@eight04 yargs can be very annoying sometimes :-)

Anyway, it looks like that we can use a combination of yargs option config properties to make it work as expected:

```js
    'ignore-files': {
      alias: 'i',
      describe: '...',
      demand: false,
      requiresArg: true,
      type: 'array',
    },
```

Basically it seems that we have to:
- omit the empty array currently set as its default value
- add a `demand: false` config to allow the --ignore-files option to be undefined (which is going to mean "that the option has not been specified") 
- add the `requiresArg: true` which force "--ignore-files" to have a value when specified

I briefly tried the above config locally and it seems to work as expected, 
let me know if it doesn't for you.